infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -66,13 +67,12 @@
             .setLength(-1)
             .build()
             .getSerializedSize();
-    public static int VERSION = 2;
-    public static int RECORDS_PER_LOG_FILE = 10000;
-    public static int TRIM_THRESHOLD = (int) (.25 * RECORDS_PER_LOG_FILE);
-    public final String logDir;
+    public static final int VERSION = 2;
+    public static final int RECORDS_PER_LOG_FILE = 10000;
+    private final String logDir;
     private final boolean noVerify;
     private final ServerContext serverContext;
-    private Map<String, SegmentHandle> writeChannels;
+    private ConcurrentMap<String, SegmentHandle> writeChannels;
```

Comment:
Why do we need a concurrent map here?
There are no parallel operations.

Reply:
It always was ConcurrentMap, I just highlighted it in an Interface. 
Not sure if I have enough confidence to change it from ConcurrentMap to a Map

Reply:
And also there are parallel operations, we use volatile variables and synchronized blocks

Reply:
I see. The `close`, however, used to reset this to HashMap.
The parallelism is serialized by the BatchWriter and the synchronized blocks. So the access to the map is always serial.
Anyways, we can leave this as is.